src/main/java/com/kk/strong/controller/UserController.java

```diff
@@ -0,0 +1,33 @@
+package com.kk.strong.controller;
+
+import com.kk.strong.model.User;
+import com.kk.strong.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @GetMapping(path = "/users")
+    public ResponseEntity<List<User>> getUsers() {
```

Comment:
You should not return entity from database in controllers, I suppose this is quick poc of security, but in the final solution you should not join database with api response -> whenever you change database or api you have to adjust the second part of the equation

Reply:
Yep, there will be a Dto for the user, this is just an example controller to test the API, will be replaced soon

Reply:
please leave a todo for that, you can always forgot about it